tests/testthat/test-recipes-nnmf_sparse.R

```diff
@@ -0,0 +1,145 @@
+library(testthat)
+library(recipes)
+library(Matrix) # Waiting for fix in RcppML
+
+test_that('Correct values', {
+  skip_if(utils::packageVersion("recipes") < "0.1.17.9001")
+
+  rec <- recipe(Species ~ ., data = iris) %>%
+    step_nnmf_sparse(all_predictors(), seed = 2432)
+
+  # dat <- Matrix::Matrix(t(as.matrix(iris[, -5])), sparse = TRUE)
```

Comment:
Since this is extratests, just use this code instead of `dput()`ing  the results in.